### ISSUE TITLE
Save Changes button not disabled when error within Keyboard Shortcuts panel

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#542]
*General Instructions*
When a DUPLICATE or INVALID ENTRY error is displayed on any text field in the Keyboard shortcuts panel, the Save Changes button is NOT disabled. Pressing Save Changes will reset the values back to the default.

#### This issue is a Bug
It could be argued that this is a feature but the expected behavior needs to be documented properly.

#### Expected Behavior
The Save Changes button should be disabled if there is an error on the current settings panel or a message should be displayed that the values were reset back to the default. 

#### Observed behavior
Readium does not disable the Save Changes button but resets the error values back to the default with no prompt or warning.

#### Steps to reproduce
1. Open the Settings Dialog and navigate to the Keyboard shortcuts pane.
2. Focus the Show Toolbar text field. 
3. Delete the value and change it to a t - DUPLICATE is displayed for the Show Toolbar and Table of Contents Field. (You could also just delete the Show toolbar value and INVALID will be displayed.)
4. Scroll down and press Save Changes button to close the Settings dialog. There is no message that the changes will not actually be saved.
5. Open the Settings Dialog and note that the Show Toolbar text field is displaying the default value of v. This is probably the right thing to do but the user
1. Either should NOT be allowed to Save the Changes when there is an error or
2. There should be a message that the changes were not saved and the values were reset to the default. Probably have to reset ALL of the settings back to the default since there could be multiple conflicts.

### Product
Readium Chrome extension

latest official version available from the Chrome Web Store - 2.23.0
Version of the Chrome browser - Version 51.0.2704.103 (64-bit)
OS and version - OS X 10.11.5

Readium cloud reader app
2.24.0-alpha
Wed, 15 Jun 2016 15:16:04 GMT
readium-js-viewer@39939d8
readium-js@b32f9eb
readium-shared-js@91324db
readium-cfi-js@a9b07a7
Browser and version - Safari Version 9.1.1 (11601.6.17)
OS and version - OS X 10.11.5
* Native application (Readium SDK C++)
  * official "launcher" sample OR propietary app
  * OS and version
* Hybrid app (ReadiumJS)
  * Electron OR Apache Cordova OR ?
  * OS and version
* Other?

**Comment by danielweck** [comment]
Good point. The invalid values are simply *not* saved, so I would prefer adding a message saying so.

**Comment by becka11y** [comment]
refs #533 - this message dialog does not trap focus

**Comment by danielweck** [comment]
Fixed by https://github.com/readium/readium-js-viewer/pull/547
